WORKSPACE

```diff
@@ -510,3 +526,12 @@ jvm_maven_import_external(
     licenses = ["notice"],  # Apache 2.0
     server_urls = ["https://repo1.maven.org/maven2"],
 )
+
+http_archive(
```

Comment:
We need to import `rules_rust` to get access to `CrateInfo` exposed by each `rust_library` and `rust_binary` target.